src/harness/fourslash.ts

```diff
@@ -699,7 +704,7 @@ module FourSlash {
             this.scenarioActions.push('<VerifyCompletionDoesNotContainItem ItemName="' + escapeXmlAttributeValue(symbol) + '" />');
 
             var completions = this.getCompletionListAtCaret();
-            if (completions && completions.entries && completions.entries.filter(e => e.name === symbol).length !== 0) {
+            if (completions && completions.entries.filter(e => e.name === symbol).length !== 0) {

```

Comment:
My understanding is `completions` can be `undefined` but `entries` is non-optional. Can @mhegazy weigh in on the accuracy of this?